### PR TITLE
Add learning outcomes readme

### PR DESCRIPTION
**Pull request by PiotrBerebecki**
List the topics required to develop a full stack app based on an Express server

relates #462

### CODE REVIEW

File: coursebook/week-8/learning-outcomes.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- Learn how to use Express middleware, for example, to serve static assets
+- Understand the purpose of templating
+- Learn how to use templating with Handlebars
+- Learn how to use Handlebars views, layouts and partials
+- Learn how to use Handlebars helpers
+- Learn how to tests an Express app
+- Learn how to include Travis CI continuous integration in an Express app
+- Learn how to integrate a PostgreSQL database with an Express app
+- Learn how to deploy an Express app to Heroku
+- Learn how to set up [an animated app drawer](http://www.material-ui.com/#/components/drawer)
```
Comment (des-des) [inline coursebook/week-8/learning-outcomes.md]: React?

Comment (PiotrBerebecki) [inline coursebook/week-8/learning-outcomes.md]: We've added a link just to see the example. The workshop will be based on vanilla JS.

Comment (des-des) [inline coursebook/week-8/learning-outcomes.md]: coolio

Comment (jsms90) [inline coursebook/week-8/learning-outcomes.md]: @des-des good to merge?

**Pull request merged**